### PR TITLE
Add coverity in readme

### PR DESCRIPTION
**Pull request by jyao1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,12 @@
 | [RISCV_XPACK](https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack) |  -   |  -  |  -  |    -    | riscv-none-elf-gcc | riscv-none-elf-gcc |
 | [RISCV_NONE](https://archlinux.org/packages/extra/x86_64/riscv64-elf-gcc/) |  -   |  -  |  -  |    -    | riscv64-elf-gcc | riscv64-elf-gcc |
 
+8) Static Analysis
+
+Support [Coverity](https://scan.coverity.com/) Scan tool.
```
Comment (steven-bellock) [inline README.md:111]: Also https://codeql.github.com/.

File: README.md [hunk 2]
```diff
@@ -106,6 +106,12 @@
 | [RISCV_XPACK](https://github.com/xpack-dev-tools/riscv-none-elf-gcc-xpack) |  -   |  -  |  -  |    -    | riscv-none-elf-gcc | riscv-none-elf-gcc |
 | [RISCV_NONE](https://archlinux.org/packages/extra/x86_64/riscv64-elf-gcc/) |  -   |  -  |  -  |    -    | riscv64-elf-gcc | riscv64-elf-gcc |
 
+8) Static Analysis
+
+Support [Coverity](https://scan.coverity.com/) Scan tool.
+
+See [nightly Coverity scanning](https://github.com/DMTF/libspdm/blob/main/.github/workflows/coverity.yml) in github CI.
```
Comment (steven-bellock) [inline README.md:113]: This has been disabled. I have to run it manually. You can delete this line.

**Review by steven-bellock — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jyao1) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (jyao1) [reply README.md]: deleted

**Review by steven-bellock — APPROVED** [review]

**Pull request merged**